package.json

```diff
@@ -29,12 +37,13 @@
     "html-tag-parser": "^1.0.0",
     "htmlparser2": "^3.10.0",
     "husky": "^1.2.0",
-    "microbundle": "^0.11.0",
```

Comment:
I didn’t want to remove microbundle at first, but the TypeScript support for microbundle has issues. I wasn’t able to get it to generate the same thing that the native TypeScript compiler did, which means that if we kept microbundle, the Node API would change. That seemed like a non-starter for me to change the API, so I simplified the bundling by using only TypeScript + Terser.